llvm/docs/GitHub.rst

```diff
@@ -23,6 +23,23 @@ intended to be able to support "stacked" pull-request. Do not create any branche
 llvm/llvm-project repository otherwise, please use a fork (see below). User branches that
 aren't associated with a pull-request **will be deleted**.
 
+Using Graphite for stacked Pull Requests
+========================================
+
+[Graphite](https://app.graphite.dev/) is a stacked pull request tool supported
```

Comment:
This seems to be the wrong markup for hyperlinks. See the rendered version: https://llvm.org/docs//GitHub.html#using-graphite-for-stacked-pull-requests

Reply:
Fixed in 62601250786244981026164b21f553c9c40a4602